gradlew.bat

```diff
@@ -1,100 +1,103 @@
-@rem
```

Comment:
Looks like new gradle wrapper version has different line ending characters for bat file.

Reply:
my bad, we should create a `.gitattributes` file with something like  
```
gradlew text eol=lf
gradlew.bat text eol=crlf
```  
this way, line endings will be OK on git pull.

Reply:
Should I amend this in that case?

Reply:
yes, thx.

Reply:
Done 👍